mad-dog/README.md

```diff
@@ -32,14 +28,15 @@ Serveral tests rely on the [`pumba`](https://github.com/alexei-led/pumba), a cha
 
 ## Notes
 - `service-commands` need to be linked, or the latest version published to npm.
+- If tests fail, sometimes running `npm i` in `/libs` and relinking packages will help. Or, bring up a fresh set of local services.
```

Comment:
why is the first statement true?

Reply:
idk this was just in here before and I moved it lower. I guess I can remove it since I'm only seeing 1 test consistently fail and the other consistently pass?

Reply:
can we change this to something like:
```
In the event that mad-dog fails due to libs errors, try running `npm i` in `/libs` and relinking packages 
```

Reply:
and also keep the second sentence too

Reply:
and also keep the second sentence too

Reply:
changed!